apis/core/v1alpha1/backupsession_helpers_test.go

```diff
@@ -35,6 +35,12 @@ func TestBackupSessionPhaseBasedOnSnapshotPhase(t *testing.T) {
 		Reason: ReasonSuccessfullyCleanedSessionHistory,
 	}
 
+	finalStep := kmapi.Condition{
```

Comment:
check if you can replace `SessionHistoryCleaned` condition with  `MetricsPushed`